### PR TITLE
Closes #6 – Report of Expensive Products

### LINKED ISSUE #6: REPORT: Expensive products
As a business aanalyst, I would like to see an HTML report showing all products priced at $1000 or more

### PR DESCRIPTION
**Pull request by gqgonzales**
## Changes

- Added app `bangazonreports` and necessary files for generating reports
- Added necessary SQL for querying database for expensive products

## Testing

Description of how to test code...

- [ ] Run migrations
- [ ] Run test suite
- [ ] Seed database with `./seed_data.sh`
- [ ] Run `python manage.py runserver`
- [ ] Navigate to `http://localhost:8000/reports/expensiveproducts` in a browser
- [ ] Confirm report contents

## Related Issues

- Fixes #6

### CODE REVIEW

File: bangazonreports/views/products/expensiveproducts.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    p.description,
+                    p.quantity,
+                    p.created_date,
+                    p.category_id,
+                    p.location,
+                    p.image_path,
+                    u.id user_id,
+                    u.first_name || ' ' || u.last_name AS full_name
+                FROM bangazonapi_product p
+                    JOIN bangazonapi_customer c ON p.customer_id = c.id
```
Comment (hannahhall) [inline bangazonreports/views/products/expensiveproducts.py]: No need to join with customer or user. Sql query should only have info about the product

**Review by hannahhall — CHANGES_REQUESTED** [review]
Looks good just a minor change to the sql query

File: bangazonreports/views/products/expensiveproducts.py (hunk 1 above)
Comment (gqgonzales) [reply bangazonreports/views/products/expensiveproducts.py]: Fixed it!

**Review by hannahhall — APPROVED** [review]
👍

**Pull request merged**